demo/java/build.gradle.kts

```diff
@@ -12,7 +12,7 @@ repositories {
 }
 
 dependencies {
-    implementation("com.github.ebraminio:astronomy:2bcbf09edd335a42e013135e04c1418d0bdcde32")
+    implementation("com.github.cosinekitty:astronomy:57a736067a2ef005c05b30d94df08d46d78d5326")
```

Comment:
I feel this isn't the best given your workflow, it could make more sense if examples was on a separate project. Alternative is to first make the main Kotlin project to create a .jar file and then use, https://stackoverflow.com/a/56557241 but somehow I liked the idea of having demo project independent from the rest of project but I will check how that can be done.